types/parent-package-json/parent-package-json-tests.ts

```diff
@@ -0,0 +1,10 @@
+import * as parent from 'parent-package-json';
+
+const hasParent = parent();
```

Comment:
It's a TypeScript feature that optional arguments can be omitted, so don't really need to test that; just test with the max number of parameters.

Reply:
Ok, I have specified the optional params in the constructor.